bin/install

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)"
+
+if [ ! -d "${__dir}/venv" ]; then
+  python3 -m venv "${__dir}/venv"
+fi
+
+source "${__dir}/venv/bin/activate"
+
+if [ ! -d "${__dir}/.config" ]; then
```

Comment:
I believe what you wanted here instead would be sg like:
```
if [ -d "${__dir}/.config" ]; then
  echo "Cannot create ${__dir}/.config file as there's already a directory with that name";
  exit 1
fi
touch .config
```

Reply:
I wanted to check if there is a .config file already or not. so if it is there already we skip it instead of exiting the script and continue with installation of packages. if we add more to this repo and more packages in future and we already have a `.config` file we never can run `bin/install` anymore

Reply:
I just checked and found out `touch` command doesn't override the content if the file already exists. I will remove the `if` part for it

Reply:
First, my suggestion actually drops an error if the user accidentally created a directory, so it may have been a better option.
Second, creating an installer only to run `python3 -m venv venv; touch .config` does not really justify its existence. May be it's a better option to put these steps into the README instead, since in this form these scripts only add complexity
What may be an option is to move the sample JSON body from the README to the `bin/install`, so the user don't have
to do it himself.

```
$ bash
$ help test|grep '\-d'
      -d FILE        True if file is a directory.

# and indeed:
$ touch .file
$ mkdir .dir
$ if [ -d .file ]; then echo OK; fi
$ if [ -d .dir ]; then echo OK; fi
OK
```

Reply:
I dropped the scripts and wrote everything in the README. I wouldn't keep a JSON creation in the installer since as you mentioned it would't justify its existence.  I think README should be okay now 🤔

Reply:
Let's resolve it, since `config.json` is unlikely to be created accidentally as a directory, also `touch` is gone.